analysis/monkmistweaver/src/modules/talents/ManaTea.tsx

```diff
@@ -21,12 +21,18 @@ class ManaTea extends Analyzer {
   manateaCount: number = 0;
   casts: Map<string, number> = new Map<string, number>();
   effectiveHealing: number = 0;
+  manaPerManaTeaGoal: number = 0;
   overhealing: number = 0;
   protected abilityTracker!: AbilityTracker;
 
   constructor(options: Options) {
     super(options);
     this.active = this.selectedCombatant.hasTalent(SPELLS.MANA_TEA_TALENT.id);
+    this.manaPerManaTeaGoal = this.selectedCombatant.hasTalent(
```

Comment:
Not to be anal but putting this after the `if (!this.active){return;}` is the better thing to do but who cares amirite